test/e2e/overrides_test.go

```diff
@@ -8,6 +8,9 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"github.com/openshift/cert-manager-operator/api/operator/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
We should add a newline here to separate the ginkgo/gomega imports from the other k8s and cert-manager imports.

Reply:
And, it would be good if k8s and our operator imports are in another block. Please refer: https://github.com/openshift/cert-manager-operator/blob/46069515b0bda3a5f51ba88705095a089258d07b/test/e2e/suite_test.go#L6-L24 for example.